edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/ModuleManagementHttpClient.cs

```diff
@@ -95,6 +99,37 @@ internal static ModuleManagementHttpClientVersioned GetVersionedModuleManagement
             return new Version_2018_06_28.ModuleManagementHttpClient(managementUri);
         }
 
+        Task Throttle(Func<Task> func) => this.Throttle<bool>(
```

Comment:
nit: we know what we are throttling, so maybe it is good to give more context to `func()` generic function name